######################
routes:
    route0:
        corridor0:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [5., 0]
            t2: [0, 0.5]
            t3: [-1., 0.]
            t4: [0, -0.5]

            waypoint: [4., 0.25, 0., 0.]
            direction: [0]

        corridor1:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [5., 0]
            t2: [0, 1.5]
            t3: [2.5, 0]
            t4: [0, -0.5]

            waypoint: [3.325, 1.2, -1.57, -1.57]
            direction: [0]

        corridor2:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [3.65, 0]
            t2: [0, 3.]
            t3: [3., 0.]
            t4: [0, -0.5]

            waypoint: [3.325, 2.75, -1.57, -1.57]
            direction: [1]

    route1:
        corridor0:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [3.65, 0]
            t2: [0, 3.]
            t3: [3., 0.]
            t4: [0, 0.25]

            waypoint: [3.325, 1., -1.57, -1.57]
            direction: [0]

        corridor1:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [5., 0]
            t2: [0, 1.5]
            t3: [-1., 0]
            t4: [0, 0.]

            waypoint: [0.5, 1., -3.14, -3.14]
            direction: [0]

        corridor2:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [2., 0]
            t2: [0, 2.]
            t3: [0.5, 0]
            t4: [0, 0.5]

            waypoint: [1.75, 1.75, -3.14, -3.14]
            direction: [1]

        corridor3:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [2.5, 0]
            t2: [0, 2.]
            t3: [1., 0]
            t4: [0, 1.5]

            waypoint: [2.25, 1.75, -3.14, -3.14]
            direction: [1]

    route2:
        corridor0:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [2.5, 0]
            t2: [0, 2.]
            t3: [0.5, 0.]
            t4: [0, -0.5]

            waypoint: [1., 1., -3.14, -3.14]
            direction: [0]

        corridor1:
            n1: [1, 0]
            n2: [0, 1]
            n3: [-1, 0]
            n4: [0, -1]

            t1: [5., 0]
            t2: [0, 1.5]
            t3: [-1., 0.]
            t4: [0, -0.5]

            waypoint: [0., 0., 0., 0.]
            direction: [0]
